### PR TITLE
Main games dev

### LINKED ISSUE #2: Can’t get out of context
During the game I can’t start another game! Must be fixed.
err

### PR DESCRIPTION
**Pull request by ExDoZu**
This pull request includes all the basic games of the bot:

- Words game
- Fill the gaps
- Word translation
- Deciphering voice
- Identify the author by a famous quote

### CODE REVIEW

**Comment by ExDoZu** [comment]
@IndianMax03 review, please!

**Comment by IndianMax03** [comment]
@ExDoZu Hey, buddy, of course I’ll do the review for you!

**Comment by IndianMax03** [comment]
1. Pls remove `poetry.lock` from PR

File: src/handlers/deciphering.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from aiogram import F, Router
+from aiogram.filters import Command
+from aiogram.types import Message, FSInputFile
+
+from aiogram.fsm.context import FSMContext
+from aiogram.fsm.state import State, StatesGroup
+
+
+from database import get_random_speech
```
Comment (IndianMax03) [inline src/handlers/deciphering.py]: 2. Use the source directory when importing the database file

**Comment by ExDoZu** [comment]
> 1. Pls remove `poetry.lock` from PR

https://python-poetry.org/docs/basic-usage/

> When Poetry has finished installing, it writes all the packages and their exact versions that it downloaded to the poetry.lock file, locking the project to those specific versions. You should commit the poetry.lock file to your project repo so that all people working on the project are locked to the same versions of dependencies.

File: src/handlers/deciphering.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class DecipheringVoice(StatesGroup):
+    answering = State()
+
+
+@router.message(Command("deciphering"))
+async def deciphering_handler(message: Message, state: FSMContext):
+
+    rowid, text = get_random_speech()
```
Comment (IndianMax03) [inline src/handlers/deciphering.py]: Rename to row_id

File: src/database.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+import logging
+
+DB_NAME = "volume/gameguage.db"
+
+
+def create_db(db_name: str = None) -> None:
+    """Create the database if it does not exist."""
+    global DB_NAME
+    if db_name is not None and db_name != DB_NAME:
+        logging.info("Replaced default database name %s with %s", DB_NAME, db_name)
```
Comment (IndianMax03) [inline src/database.py:12]: Python string with `%s` is an old string formatting technique. Use fstring instead.

**Review by IndianMax03 — COMMENTED** [review]
I added a few corrections. Thank you very much for your work!

File: src/database.py (hunk 3 above)
Comment (ExDoZu) [reply src/database.py:12]: This is not outdated. This is an optimization feature

File: src/handlers/deciphering.py (hunk 2 above)
Comment (ExDoZu) [reply src/handlers/deciphering.py]: fixed

File: src/handlers/deciphering.py (hunk 1 above)
Comment (ExDoZu) [reply src/handlers/deciphering.py]: fix

Comment (IndianMax03) [reply src/handlers/deciphering.py]: Great job!

File: src/handlers/deciphering.py (hunk 2 above)
Comment (IndianMax03) [reply src/handlers/deciphering.py]: Excellent!

File: src/database.py (hunk 3 above)
Comment (IndianMax03) [reply src/database.py:12]: Oh, ofc, I’m sorry for not paying attention!

File: README.md
Comment (Anth0o0ny) [inline README.md:1]: Ненужное уточнение

**Review by Anth0o0ny — COMMENTED** [review]
great job, you can merge

**Review by IndianMax03 — APPROVED** [review]
Approve!

**Review by Anth0o0ny — APPROVED** [review]
great job

**Pull request merged**